Documentation/En/Index.xyl

```diff
@@ -77,25 +77,25 @@
   <p>The “Conception support” axis describes the <strong>visibility</strong> we
   have on the System Under Test: Either <strong>black box</strong> or
   <strong>white box</strong>. Their meanings are different according to the
-  context but we can consider a system as a white box if we look and use what
-  is <strong>inside</strong> the box, else this is a black box. For instance,
-  for unit tests, let say with a function as the SUT, black box testing will
+  context but we can consider a system as a white box if we look at and use what
+  is <strong>inside</strong> the box, else it is a black box. For instance,
+  with unit tests, lets say with a function as the SUT, black box testing will
   consider only the inputs and the outputs of the function without considering
   its body (even if it reads or writes global variables, streams etc.). On the
   other hand, white box testing will consider the body of the function and the
   testing strategy will be totally different.  What we expect from a black box
   or a white box system is not the same.</p>
   <p>The “System size" axis describes at what <strong>level</strong> (or
-  altitude) do we test: From the lowest one, <strong>unit</strong> (a function
-  or a method) to the highest one, the whole <strong>system</strong>, which
-  includes <strong>components</strong> (a set of functions) and the
-  <strong>integration</strong> of these components inside other ones, which
-  together form the system. All levels are important: This is not because a
-  function does what it is supposed to do that it will integrate nicely with
-  another one.</p>
+  altitude) the system will be tested: From the lowest one,
+  <strong>unit</strong> (a function or a method) to the highest one, the whole
+  <strong>system</strong>, which includes <strong>components</strong> (a set of
+  functions) and the <strong>integration</strong> of these components inside
+  other ones, which together form the system. All levels are important: This is
+  not because a function does what it is supposed to do, but that it will integrate
```

Comment:
ok, I think I read it correctly now - if I substitute "This is not because" with "It is not because" then it makes sense somehow).

````
All levels are important: It is not that a function does what it is supposed to do,
but that it integrates correctly with others.
````

But even then, it seems to suggest that Unit testing is less important. I would say that Unit Testing is the most important level, as in addition to allowing you to test all the code paths in a Unit, it also informs good design. Having only functional tests in a library ensures that the system works, but ignores its architecture. For me it is important *both* that a Unit does what it is supposed to do and that it integrates well with others.